assets/js/utils/rtcUtils.ts

```diff
@@ -173,10 +173,7 @@ export const shareScreen = async (
 
     if (!webrtc) return peersState; // If webrtc is null, we only add media source locally
 
-    const newTrack = findTrackByType(client, "video", peersState);
-    if (!newTrack) return peersState;
```

Comment:
Doesn't it break anything? Because previously if it wasn't a new track we return from this scope, currently we only update peerState.